app/blueprints/v1/app_blueprint.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module V1
+  class AppBlueprint < Blueprinter::Base
+    fields :id, :name
```

Comment:
It would be nice to add the `created_at` field.

Reply:
done